server/views/partials/personalPage/neurodiversity.njk

```diff
@@ -1,16 +1,87 @@
 {% from "../../macros/summaryCardMacro.njk" import summaryCard %}
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "../../macros/summaryListRow.njk" import summaryListRow %}
 {%- call summaryCard({title: "Neurodiversity", id: "neurodiversity"}) -%}
-  <div class="govuk-grid-row" data-qa="personal-details">
+  <div class="govuk-grid-row" data-qa="neurodiversity">
     <div class="govuk-grid-column-full">
-      <p class="govuk-body">
-        DPS currently includes neurodiversity information recorded for prisoners in HMPs Bristol, Lincoln, New Hall and
-        Swaleside.
-      </p>
-      <p class="govuk-body">
-        To check for neurodiversity information for someone in another prison, contact the education team in that
-        prison.
-      </p>
+      {% if not learnerNeurodivergence.length %}
+        <p class="govuk-body">
+          DPS currently includes neurodiversity information recorded for prisoners in HMPs Bristol, Lincoln, New Hall and
+          Swaleside.
+        </p>
+        <p class="govuk-body">
+          To check for neurodiversity information for someone in another prison, contact the education team in that
+          prison.
+        </p>
+      {% elseif learnerNeurodivergence.length %}
+        <p class="govuk-body" data-qa="neurodivergence-exists">
+          Information from the local education team - not a medical diagnosis.
+        </p>
+
+        {# Support needed #}
+        {% if learnerNeurodivergence[0].neurodivergenceSupport.length %}
```

Comment:
Not sure what the actual data looks like, but should this always be hard-coded to the first entry in the array?

Can there be multiple entries? And if so, do they all need to be displayed, or only (latest?) one? What order does the array return entries?

Reply:
Thanks for taking a look, yes all the curious API responses are arrays which contain single objects.
[https://editor.swagger.io/?url=https://raw.githubusercontent.com/ministryofjustice/curious-API/main/curious-api-specification.yaml](url)